warehouse/templates/api/simple/detail.html

```diff
@@ -15,6 +15,9 @@
 <html>
   <head>
     <meta name="pypi:repository-version" content="{{ meta['api-version'] }}">
+    {% for alt_repo in alternate_locations %}
+    <meta name="pypi:alternate-locations" content="{{ alt_repo }}">
```

Comment:
Note: Text of PEP says "... in HTML as a meta element named pypi-alternate-locations..." while the example shows `pypi:alternate-locations`.

@dstufft I'm assuming that consistency with `pypi:repository-version` is intended, so the PEP should likely be updated for consistency.